src/lib/openapi/spec/project-schema.ts

```diff
@@ -55,6 +55,14 @@ export const projectSchema = {
             description:
                 '`true` if the project was favorited, otherwise `false`.',
         },
+        mode: {
+            type: 'string',
+            enum: ['open', 'protected'],
+            example: 'open',
+            nullable: true,
+            description:
+                'A mode of the project affecting what actions are possible in this project.',
```

Comment:
Can we add to the description what it means when the property is `null` or missing? So that consumers don't have to wonder?

```suggestion
                'A mode of the project affecting what actions are possible in this project.',
```

Reply:
What documentation strategy do we have for experimental/WIP capabilities?

Reply:
Also we don't have this type of information for all other fields in this schema and they are mostly optional.

Reply:
Added extra info about a rollout of a new feature so that people on the bleeding edge are not confused about this field

Reply:
Great! So what you're saying (if I understand the changes in d87a8e8205c6245486aeee0494255237df3a2219 correctly), is that this will only be `null` during rolling out this feature? Also the description doesn't say anything about when it's undefined. What does that mean?

And you're right. The other fields don't have this info today, but that doesn't mean that they can't be better. We should strive to explain all the fields completely to the end user. That can either be on the property-level or on the schema-level.